sga/urls.py

```diff
@@ -1,11 +1,23 @@
-"""
-URLs for sga
-"""
-from django.conf.urls import include, url
-from sga.views import index
+from django.conf.urls import url
+from django.conf.urls.static import static
+
+from sga.views import index, dev_start, view_submission_as_student, view_student_list, \
+    view_assignment, view_submission_as_staff, view_assignment_list, view_student, \
+    view_grader_list, view_grader, unsubmit_submission, change_student_to_grader, \
+    change_grader_to_student, download_all_submissions, download_not_graded_submissions, unassign_grader, \
+    unassign_student
```

Comment:
multiple imports should be done in this way:

```
from sga.views import (
    index,
    dev_start,
    view_submission_as_student,
    ...
)
```

in your case, considering the amount of imports, it makes much more sense to import the module
```
from sga import views
```